src/registrar/migrations/0049_alter_transitiondomain_status.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.2.7 on 2023-12-01 21:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("registrar", "0048_alter_transitiondomain_status"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="transitiondomain",
+            name="status",
+            field=models.CharField(
+                blank=True,
+                choices=[("ready", "Ready"), ("on hold", "On hold"), ("unknown", "Unknown")],
+                default="ready",
+                help_text="domain status during the transfer",
+                max_length=255,
+                verbose_name="Status",
+            ),
+        ),
+    ]
```

Comment:
Would it be better to modify the last instance of this rather than creating a new migration file? Not sure if that'd break anything on stable, which is my main thought with that